Gemfile

```diff
@@ -99,12 +66,126 @@ gem "ruport",                         "=1.7.0",                       :git => "h
 # https://github.com/jeremyevans/ruby-american_date
 gem "american_date"
 
-group :ui_dependencies do # Added to Bundler.require in config/application.rb
+# Make sure to tag your new bundler group with the manageiq_default group in addition to your specific bundler group name.
+# This default is used to automatically require all of our gems in processes that don't specify which bundler groups they want.
+#
+### providers
+group :amazon, :manageiq_default do
+  manageiq_plugin "manageiq-providers-amazon"
+end
+
+group :ansible, :manageiq_default do
+  gem "ansible_tower_client",           "~>0.12.2",      :require => false
+end
+
+group :azure, :manageiq_default do
+  manageiq_plugin "manageiq-providers-azure"
+end
+
+group :foreman, :manageiq_default do
+  manageiq_plugin "manageiq-providers-foreman"
+  gem "foreman_api_client",             ">=0.1.0",   :require => false, :git => "https://github.com/ManageIQ/foreman_api_client.git", :branch => "master"
```

Comment:
Sure... that's outside the scope of this PR.  I'm not sure why we're riding master here.  Lots of the "bundler groups" have these developer friendly things that hopefully are easier to see with bundler groups.  We can fix them one by one.